openml/flows/functions.py

```diff
@@ -1,38 +1,30 @@
 import xmltodict
 
 from openml._api_calls import _perform_api_call
+# Absolute imports, to avoid circular dependencies
+from openml.flows.sklearn_converter import flow_to_sklearn
```

Comment:
That doesn't seem right. Can you explain? I'm pretty sure that won't fix any circular import issues, and it can lead to weird stuff..